subworkflows/local/mirna_quant.nf

```diff
@@ -15,8 +15,8 @@ include {   BOWTIE_MAP_SEQ  as BOWTIE_MAP_MATURE
             BOWTIE_MAP_SEQ  as BOWTIE_MAP_HAIRPIN
             BOWTIE_MAP_SEQ  as BOWTIE_MAP_SEQCLUSTER } from '../../modules/local/bowtie_map_mirna'
 
-include {   BAM_SORT_SAMTOOLS as BAM_STATS_MATURE
-            BAM_SORT_SAMTOOLS as BAM_STATS_HAIRPIN   } from '../nf-core/bam_sort_samtools'
+include {   BAM_SORT_STATS_SAMTOOLS as BAM_STATS_MATURE
```

Comment:
The second argument of these calls below needs to be a map like `[meta, fasta]. Maybe the easiest solution would be to also adapt the module `format_fasta_mirna` to also accept and return a map of `[meta, fasta]`.